website/scripts/python-doc-gen.sh

```diff
@@ -20,9 +20,10 @@ set -e
 HERONPY_VERSION=$1
 HERON_ROOT_DIR=$(git rev-parse --show-toplevel)
 INPUT=heronpy
-TMP_DIR=$(mktemp --directory)
+TMP_DIR=$(mktemp -d)
 
-VENV="$(mktemp --directory)"
+VENV="$(mktemp -d)"
+pip install virtualenv
```

Comment:
Should we check if the `virtualenv` is already installed before using `pip install`?

Reply:
I think pip doesn't reinstall package if it is already installed.